esp/src/eclwatch/PackageMapPartsWidget.js

```diff
@@ -93,9 +93,8 @@ define([
                     var addPartInput = context.getFilter();
                     WsPackageMaps.AddPartToPackageMap({
                         request: {
-                            Target: addPartInput.AddPartsTarget,
-                            Process: addPartInput.Process,
-                            PackageMap: addPartInput.PackageMap,
+                            Target: context.params.target,
+                            PackageMap: context.params.packageMap.split('::')[1],
```

Comment:
@afishbeck Can you verify? The package map from the previous screen is always separated by '::' so I am trimming to pass it in the request.